server/resources/installation-notes.html

```diff
@@ -57,6 +57,17 @@ <h3>Important Note with regards to ICU</h3>
 
 <p>PostgreSQL Server 17 is compiled with ICU 67.x. The server versions upto v12 used either ICU 53.x or  57.x. Hence, if you use pg_upgrade for a major version upgrade and have ICU collated indexes, then you need to reindex the indexes.</p>
 
+<h3>Important Note with regards to locale names</h3>
+
+<p>Windows installers for PostgreSQL versions 17.4-1, 16.8-1, 15.12-1, 14.17-1, and 13.20-1, convert locale names to BCP-47 compulsorily before passing on to initdb. So, long name locales are not compatible in these installers.</p>
```

Comment:
A change was made in the Windows installers for PostgreSQL versions 17.4-1, 16.8-1, 15.12-1, 14.17-1, and 13.20-1, and in the fresh installation the chosen locale name would be converted to BCP-47 name and passed to initdb.exe run as --locale. This change made the pg_upgrade from the PostgreSQL cluster having old locale names to these versions fail.

The change was again made in -2 releases for the same set of postgresql installers. See below section for details